cmd/gb-vendor/fetch.go

```diff
@@ -70,4 +99,5 @@ var FetchCmd = &cmd.Command{
 		}
 		return wc.Destroy()
```

Comment:
Cool. I can see the direction your going so there is no need for extended
debate in PRs. If we stick to designing features up front in issues then
small PRs will work out fine.

Thanks

Dave

On Wed, 27 May 2015 17:10 Dávid Kaya <notifications@github.com> wrote:

> In cmd/gb-vendor/fetch.go
> <https://github.com/constabulary/gb/pull/134#discussion_r31108152>:
>
> > @@ -70,4 +99,5 @@ var FetchCmd = &cmd.Command{
> >  		}
> >  		return wc.Destroy()
>
> Ok, I will create smaller PRs for feature changes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/constabulary/gb/pull/134/files#r31108152>.
>